cmake/zlib.cmake

```diff
@@ -5,29 +5,29 @@ find_package(ZLIB QUIET)
 
 # If not found, build from sources
 if(NOT TARGET ZLIB::ZLIB)
-	
+	cmake_policy(SET CMP0074 NEW)
```

Comment:
Could you detail what is the purpose of this new line? Thanks

Reply:
If sets the following policy, without it this won't run on newer Cmake versions ^^

https://cmake.org/cmake/help/latest/policy/CMP0074.html

Reply:
In this case it should be on the main CMakeLists.txt to be applied to all the dependencies instead of just zlib no?

Reply:
Probably, even tho I didn't experience any issues with it not being set

Reply:
Yeah but depending on what's already on your computer it could break. For example if you have zlib but not openssl, you would not set the policy as this if would never be entered.

So if you can just move this line somewhere around [here](https://github.com/adepierre/Botcraft/blob/74ca36f8e5bcc618d4e6da19503f23a0e5d290c5/CMakeLists.txt#L6) and then I'll merge the PR to fix the issue 🙂